apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/TimedBufferingConsumer.cs

```diff
@@ -57,8 +56,9 @@ internal TimedBufferingConsumer(IConsumer<T> consumer, TimeSpan waitTime, IThrea
         /// </summary>
         /// <param name="consumer">The consumer to flush to. Must not be null.</param>
         /// <param name="waitTime">The amount of time between automatic flushes.</param>
-        public static TimedBufferingConsumer<T> Create(IConsumer<T> consumer, TimeSpan waitTime)
-            => new TimedBufferingConsumer<T>(consumer, waitTime, new SimpleThreadingTimer());
+        /// <param name="timerExceptionHandler">The action to handle exceptions thrown within the timer thread. May be null.</param>
+        public static TimedBufferingConsumer<T> Create(IConsumer<T> consumer, TimeSpan waitTime, Action<Exception> timerExceptionHandler)
```

Comment:
Can we make this method internal instead of public? That'll make it clearer for future changes that this *isn't* a breaking change (because the type is internal)

Reply:
Yes, done.